packages/genesys-spark-components/src/genesys-spark-utils/intl.ts

```diff
@@ -53,6 +53,28 @@ export function relativeTimeFormat(
   }
 }
 
+function closestPassShadow(
```

Comment:
Have a look at genesys-spark/packages/genesys-spark-components/src/utils/dom/get-closest-element.ts and see if that can be used centralised etc...